### PR TITLE
Views

### PR DESCRIPTION
**Pull request by levy002**
In this PR, I:
 

- [x] added controllers functionalities
- [x] added all views as required.

Cheers!!!!

### CODE REVIEW

File: app/views/users/show.html.erb [hunk 1]
```diff
@@ -1 +1,32 @@
...
+
+
+<div>
+    <% @user.recent_posts.each do |post| %>
+     <div>
+         <p> Post #<%= post.id %> </p>
+         <p> <%= post.text %> </p>
+     </div>
+    
+    <% end %>
```
Comment (enkog) [inline app/views/users/show.html.erb]: - [x] Please also include `comments_counter` and `likes_counter` on `views/users/show.html.erb` to ensure you have all the fields as indicated in the wireframe.

File: app/views/posts/show.html.erb [hunk 2]
```diff
@@ -1,2 +1,27 @@
...
+           <span>
+             <%= comment.author.name %>:
+            </span> 
+           <span>
+             <%= comment.text %>
+           </span>
+        </p>
+     </div>
+    
+    <% end %>
```
Comment (enkog) [inline app/views/posts/show.html.erb]: - [x] I recommend you use `post.recent_comments` to render 5 most recent comments to meet the project requirement.

***Hint:***
```ruby
<% post.recent_comments.each do |comment| %>
  <p>
    <%= comment.author.name %>: 
    <%= comment.text %>
  </p> 
 <% end %>
```

**Review by enkog — CHANGES_REQUESTED** [review]
Hi @levy002 👋, 

Good job so far! 👍 

✅ Great work in making use of partials for your posts and users views.
✅ Correct GitFlow
✅ Linters set up correctly
✅ Descriptive README

There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
- [ ] Please layout your pages to match the given wireframe as indicated [here](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Views_project.md).
- This is an important requirement for this project

Example: 

users/show

_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Deepakdanger — APPROVED** [review]
Hi @levy002, 

### Status: Approved ✔️

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚢 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by levy002** [comment]
Thank you @Deepakdanger :handshake:

**Pull request merged**